ecl/eclcc/eclcc.cpp

```diff
@@ -929,7 +929,8 @@ void EclCC::instantECL(EclCompileInstance & instance, IWorkUnit *wu, const char
                     }
                     else
                     {
-                        generator->addArchiveAsResource(buf);
+                        if (!instance.wu->getDebugValueBool("obfuscateOutput", false))
```

Comment:
This change prevents the archive being stored if supplied as a command line option